bloom-housing/bloom
# 1655/invite user edit

# Pull Request Template

## Issue

Addresses #1655


## Description
Adds the ability to edit a user, delete and resend an invite.

## Type of change


## How Can This Be Tested/Reviewed?

1. Create a new user, activate it using a link from the email, and try to log in
2. Create a user and do not activate it. Instead of - click on the "resend confirmation" button and activate using a new link
2. Update user data and check the user summary again
3. Delete a user and check the users table 

## Checklist:
